libevmasm/GasMeter.h

```diff
@@ -221,7 +221,7 @@ class GasMeter
 
 	/// @returns gas costs for simple instructions with constant gas costs (that do not
```

Comment:
Is this comment still correct? I mean we have now the EVM version as a parameter here - so something may be different based on that information, right?

Reply:
Ah ok, maybe not - gas costs seem to be still the same